packages/vite/src/node/plugins/define.ts

```diff
@@ -49,15 +48,12 @@ export function definePlugin(config: ResolvedConfig): Plugin {
       importMetaKeys[`import.meta.env.${key}`] = JSON.stringify(env[key])
     }
     Object.assign(importMetaFallbackKeys, {
-      'import.meta.env.': `({}).`,
       'import.meta.env': JSON.stringify(config.env),
```

Comment:
This change is fine as esbuild will share a single reference to `import.meta.env` object as a variable, so it shouldn't increase the bundle size by much after bundling